packages/ens-deployments/src/lib/types.ts

```diff
@@ -52,35 +47,37 @@ export type SubregistryContractConfig =
     };
 
 /**
- * Encodes the deployment of a subregistry, including the target chain and contracts.
+ * A DeploymentDatasource describes a set of contracts on a given chain that extend the ENS root.
+ *
+ * NOTE: this currently encodes the assumption that a given on-chain ENS datasource correlates to
```

Comment:
```suggestion
 * NOTE: this currently encodes a simplified assumption that a given onchain ENS datasource correlates to
```

Suggest we not hyphenate onchain or offchain